content/blog/fs-advent-2021-fantomas-daemon/index.md

```diff
@@ -0,0 +1,100 @@
+---
+path: '2021/12/07/fs-advent-2021-fantomas-daemon/'
+date: '2021-12-07'
+title: 'Fantomas Daemon'
+tags: ['open-source', 'fsharp', 'fantomas', 'tooling' ]
+cover: './nojaf.com.fantomas-daemon-fsadvent-2021.jpg'
+---
+
+## Intro
+
+The F# advent calendar is a wonderful initiative that inspires the community to create new content in a short period of time.
+I've been fortunate enough to be part of this for some years now. This year, I had in mind to sit this one out actually.
+Sometimes, there are just no new tales to tell and it is better suited to give others an opportunity.
+When I noticed that [Sergey still had some spots left to fill](https://twitter.com/sergey_tihon/status/1459794038115155969?s=20), I did happily volunteer, thought didn't think it through that much.
+
+My original thoughts for this post, was to create some content around the F# compiler. 
```

Comment:
"My original thought for this post was to create..." (singular "thought" and take out the comma)